src/app.tsx

```diff
@@ -47,11 +47,15 @@ export interface FolderFileInfo extends FileInfo {
     to: string | null,
 }
 
+interface FilesContextType {
+    addAlert: (title: string, variant: AlertVariant, key: string) => void,
+    rootInfo: FileInfo | null,
+}
+
 export const FilesContext = React.createContext({
-    // eslint-disable-next-line no-unused-vars
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    addAlert: (title: string, variant: AlertVariant, key: string) => console.warn("FilesContext not initialized")
-});
+    addAlert: () => console.warn("FilesContext not initialized"),
```

Comment:
Hmm, the old signature made more sense to me as documentation. Perhaps at least add a comment how it is meant to be called?

Reply:
Sure, but I got a bit tired of the copy pasting of typing.